PDP11/pdp11_rq.c

```diff
@@ -248,36 +248,60 @@ struct rqpkt {
 
 /* The RQDX3 supports multiple disk drive types (x = not implemented):
 
-   type sec     surf    cyl     tpg     gpc     RCT     LBNs
-        
-   RX50 10      1       80      5       16      -       800
-   RX33 15      2       80      2       1       -       2400
-   RD51 18      4       306     4       1       36*4    21600
-   RD31 17      4       615     4       1       3*8     41560
-   RD52 17      8       512     8       1       4*8     60480
-   RD32 17      6       820     6       1       4*8     83204
-x  RD33 17      7       1170    ?       ?       ?       138565
-   RD53 17      8       1024    8       1       5*8     138672
-   RD54 17      15      1225    15      1       7*8     311200
+   type  sec  surf  cyl   tpg  gpc  RCT   LBNs    notes
+   RX18  9    1     40    9    1    -     360
+   RX50  10   1     80    5    16   -     800
+   RX33  15   2     80    2    1    -     2400
+x  RD50  17   4     153   ?    ?    ?     ?       Seagate ST-506
+   RD51  18   4     306   4    1    36*4  21600   Seagate ST-412
+   RD31  17   4     615   4    1    3*8   41560   Seagate ST-225
+   RD52  17   8     512   8    1    4*8   60480   Quantum 540, ATASI 3046, Evotek ET3540

```

Comment:
I didn't know the RD50 is supported by MSCP.  It's a Pro device, yes, though there it has 16 sectors per track (same goes for the other RD series drives the Pro supports, because its controller is limited to a 4-bit sector number).
The parameters for RD52 are the ones for the Quantum drive; the Atasi has a different geometry (7 surfaces, 644 cylinders, don't know the sector/track count on MSCP).  And I've heard of a CDC drive DEC sold as RD52, or perhaps only planned to, that one has 5 surfaces).

Reply:
As far as I can tell, the RD50/Seagate ST-506 is not implemented in MSCP spec (it doesn't appear in the Disk Drive/Media Identifier table or the Unannounced Disk Drive/Media Identifier table). I only include it in the reference table since it is an RD-series disk though not MSCP attached (theoretically if one wanted to add it, one could; just being mindful that the model byte and media identifier code would be fictitious, likely similar to how a third-party MSCP-based controller would report its attached drives).

With regards to the RD52, I have seen the RD52 be referred to as one of the three drives listed, and the ZRQCH0 formatting program in XXDP does explicitly list two different configurations of RD52:
- RD52 part # 30-21721-02 (1 light on front panel)
- RD52 part # 30-23227-02 (2 lights on front panel)
It's possible one of those refers to the Quantum drive, and the other to the ATASI (or Evotek) drive.